local-dev/run_local.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+psql -f local-dev/local-postgres-init.sql
```

Comment:
Isn't this just using/requiring you have psql running locally with default access? I see that's what data-repo and workspace manager are doing, but running postgres with docker like Sam seems like a better idea. 
Maybe something like https://github.com/broadinstitute/sam/blob/develop/docker/run-postgres.sh ?

Reply:
Checked with Zarc about that. The reason we have the local postgres running is to have the local version of WM with the data. And that is why they have two db, testdb and wmdb. 
Testdb is for unit test and they will reset the db when server starts.
 But for the wm db, that is for local running, and data will not be reset by default. So whenever you need the local WM, you starts it and connect to the db in your local, you will have all the data. 
The SAM's one is better if we don't want to keep those databases and data? Let me change to that one. Thanks